src/main/webapp/app/shared/notification/notification-sidebar/notification-sidebar.component.ts

```diff
@@ -107,13 +108,25 @@ export class NotificationSidebarComponent implements OnInit {
 
     private loadNotificationsSuccess(notifications: Notification[], headers: HttpHeaders): void {
         this.totalNotifications = Number(headers.get('X-Total-Count')!);
-        this.addNotifications(notifications);
+        this.addNotifications(this.filterLoadedNotifiactions(notifications));
         this.page += 1;
         this.loading = false;
     }
 
+    // filter out every exam related notification
+    private filterLoadedNotifiactions(notifications: Notification[]): Notification[] {
+        // tslint:disable-next-line:triple-equals
```

Comment:
why disable linter here?

Reply:
Same problem. Look at my explanation above. (If you know a way to compare enums and strings with === please tell me)